seqstats/seqstats.go

```diff
@@ -0,0 +1,109 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// seqstats calculates and prints sequence statistics from a
+// multi-FASTA DNA sequence file (default stdin). It is useful for
+// analyzing metrics of microbial genome assemblies or metagenome
+// "bins". It prints: the total no. of sequences, assembly size
+// (total length of all sequences), Min, Max, Avg and N50.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+)
+
+// binStats contains the basename of the file without any extension and
+// other reported statistics in bp (base pairs)
+type binStats struct {
+	Name    string // from input filename (empty, if stdin)
+	totSeqs int
+	Size    int
+	Min     int
+	Max     int
+	Avg     int
+	N50     int
+}
+
+var (
+	inf  = flag.String("inf", "", "input contig file, defaults to stdin")
```

Comment:
Please change the flag string "inf" to "in". The name "inf" is already heavily loaded.